vacationtime-solution/src/Code.js

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019 Google LLC
```

Comment:
Let's add an onFormSubmit that notifies the manager there is a new vacation request